modules/luci-mod-system/luasrc/model/cbi/admin_system/system.lua

```diff
@@ -105,6 +106,29 @@ o:value(8, translate("Normal"))
 o:value(9, translate("Warning"))
 
 
+--
+-- Zram Properties
+--
+if has_zram then
+	o = s:taboption("zram", Value, "zram_size_mb", translate("ZRam Size"), "Mb")
```

Comment:
Use "MB" please (or is it indeed Mega*bit*? In this case abbreviate it as `MBit`)

Also given that this is the description string, maybe be a bit more verbose, like `Size of the ZRam device in megabytes`